examples/values/backend-org-1.yaml

```diff
@@ -42,7 +42,7 @@ containerRegistry:
     - image: substra/substra-tools:latest
       sourceRegistry: ghcr.io
       dstImage: substra/substra-tools:latest
-
+  userImageRepository: substra-user-images
```

Comment:
I think we want to add the parameter explicitly in all orgs, for clarity.

Reply:
I also changed the default to `substra/user-image` to remove the `substrafoundation` mention out of date. But I let the org1 to `substra-user-images` to continue to test that the feature works